predicators/spot_utils/perception/object_detection.py

```diff
@@ -147,7 +146,7 @@ def detect_objects_from_april_tags(
 def detect_objects_from_language(
     object_ids: Collection[LanguageObjectDetectionID],
     rgbds: Dict[str, RGBDImageWithContext],
-) -> Tuple[Dict[ObjectDetectionID, math_helpers.SE3Pose], Dict[str, Any]]:
+) -> Tuple[Dict[ObjectDetectionID, math_helpers.SE3Pose], Dict]:
```

Comment:
why delete the `[str, Any]` part?

Reply:
I wanted to use ObjectDetectionIDs as keys